docs/python/how-to-set-activity-retry-options-in-python.md

```diff
@@ -0,0 +1,16 @@
+---
+id: how-to-set-an-activity-retry-policy-in-python
+title: How to set an Activity Retry Policy in Python
+sidebar_label: Retry Policy
+description: Create an instance of an Activity Retry Policy in Python.
+tags:
+  - python
+  - how-to
+---
+
+To create an Activity Retry Policy in Python, set the [RetryPolicy](https://python.temporal.io/temporalio.common.retrypolicy) class within the [`start_activity()`](https://python.temporal.io/temporalio.workflow.html#start_activity) or [`execute_activity()`](https://python.temporal.io/temporalio.workflow.html#execute_activity).
```

Comment:
```suggestion
To create an Activity Retry Policy in Python, set the [RetryPolicy](https://python.temporal.io/temporalio.common.retrypolicy) class within [`start_activity()`](https://python.temporal.io/temporalio.workflow.html#start_activity) or [`execute_activity()`](https://python.temporal.io/temporalio.workflow.html#execute_activity).
```
Alternatively, "...within the `start_activity()` or `execute_activity()` function."